OpenSim/Simulation/Model/ExpressionBasedCoordinateForce.cpp

```diff
@@ -26,6 +26,8 @@
 //=============================================================================
 #include "ExpressionBasedCoordinateForce.h"
 #include <OpenSim/Simulation/Model/Model.h>
+#include <Vendors/lepton/include/lepton/Parser.h>
+#include <Vendors/lepton/include/lepton/ParsedExpression.h>
```

Comment:
While this works, [I think you should be able to remove the `Vendors/`](https://github.com/opensim-org/opensim-core/blob/master/cmake/OpenSimMacros.cmake#L233).